exampleCourse/questions/downloadFile/question.html

```diff
@@ -0,0 +1,8 @@
+<pl_question_panel>
+    <ul>
+        <li>This text file comes from <code>clientFilesCourse</code>: <pl_file_download file_name="data.txt" type="static" scope="course" /></li>
```

Comment:
I think having both `type` and `directory` is good. Is it too confusing to have:
* The default `type` is `static`
* If `type=static` then the default `directory` is `clientFilesQuestion`
* If `type=dynamic` then it's an error to also provide a `directory`. If we want to later allow dynamic files at the course level then we could allow `directory=serverFilesCourse`? In any case we don't need to decide this now.